addons/hr_expense/static/src/js/expense_views.js

```diff
@@ -87,7 +87,9 @@ odoo.define('hr_expense.expenses.tree', function (require) {
         }
     };
 
-    var ExpenseListRenderer = ListRenderer.extend(ExpenseDashboardMixin, ExpenseQRCodeMixin);
+    var ExpenseListRenderer = ListRenderer.extend(ExpenseQRCodeMixin);
```

Comment:
I would add some comments to describe which class do what. I know this file is quite frugal with the comments but as there are many overrides, it would help people asking themselves what these classes are doing.